src/nxdoc/web-ui/web-ui-overview.md

```diff
@@ -106,41 +105,39 @@ version_override:
+
+<div class="row"><div class="column medium-4">{{#> panel type='primary'}}
 
-<div class="row">
-<div class="column medium-4" align="center">
-{{#> panel type='primary'}}
 [Polymer Guide]({{page page='polymer-guide'}})
-{{/panel}}
-</div>
-<div class="column medium-4" align="center">
-{{#> panel type='primary'}}
+
+{{/panel}}</div><div class="column medium-4">{{#> panel type='primary'}}
+
 [Nuxeo Elements]({{page page='nuxeo-elements'}})
-{{/panel}}
-</div>
-<div class="column medium-4" align="center">
-{{#> panel type='primary'}}
+
+{{/panel}}</div><div class="column medium-4">{{#> panel type='primary'}}
+
 [Customization]({{page page='web-ui-customization'}})
-{{/panel}}
-</div>
-</div>
+
+{{/panel}}</div></div>
 
 {{#> callout type='info' heading='Nuxeo University'}}
-Watch the related courses on Nuxeo University:</br>
-[Course on the Web UI Stack](https://university.nuxeo.com/learn/public/course/view/elearning/76/nuxeo-web-ui-stack-introduction).</br>
-[Course to Start with Web UI](https://university.nuxeo.com/learn/public/course/view/elearning/92/getting-started-with-nuxeo-web-ui).
+Watch the related courses on Nuxeo University
+- [Course on the Web UI Stack](https://university.nuxeo.com/learn/public/course/view/elearning/76/nuxeo-web-ui-stack-introduction).
+- [Course to Start with Web UI](https://university.nuxeo.com/learn/public/course/view/elearning/92/getting-started-with-nuxeo-web-ui).
 ![]({{file name='university-webui.png' page='nxdoc/university'}} ?w=450,border=true)
 {{/callout}}
 
 ## Requirements
 
 {{! multiexcerpt name='webui-requirements'}}
-Nuxeo Web UI and Nuxeo Web UI Elements framework make use of the recent W3C standard Web Components and Google Polymer 2.0 framework (legacy mode). The Google Polymer framework comes with polyfills for Web Components, a library that extends the support of web components standard to more browsers.
+Nuxeo Web UI and Nuxeo Web UI Elements framework make use of the recent W3C standard Web Components and Google Polymer 3.0 framework (legacy mode). The Google Polymer framework comes with polyfills for Web Components, a library that extends the support of web components standard to more browsers.
 
 {{#> callout type='warning' heading='Polymer 2.0 legacy mode'}}
-Previous versions of Nuxeo Web UI (1.x) relied on Polymer 1 so for compatibility reasons and to ensure a simple incremental upgrade the current version of Nuxeo Web UI is running
-Polymer 2 in legacy mode thus using the Polymer factory method, not the ES6 class-based syntax. Most of the 1.0 APIs are thus still available, as are any new 2.x APIs.
+
+Previous versions of Nuxeo Web UI relied on Polymer version 1 or 2, so for compatibility reasons and to ensure a simple incremental upgrade the current version of Nuxeo Web UI is running
+Polymer 3 in legacy mode thus using the Polymer factory method, not the ES6 class-based syntax. Most of the 2.0 APIs are thus still available, as are any new 2.x APIs.
```

Comment:
Some elements on Web UI already use ES6 and classed based syntax, so we need to re-write this. 

We can say that Web UI runs in polymer 3 but we keep compat with previous APIs that allow both Polymer 1 and Polymer 2 elements to work with it. And we can reference the upgrade notes page here,